roles/ec2_provision/defaults/main.yml

```diff
@@ -3,3 +3,5 @@ ec2_region: "{{ lookup('ENV', 'EC2_REGION') or 'us-east-2' }}"
 ec2_user: "{{ ansible_user }}"
 nfs_cidr: "10.1.0.0/16"
 nfs_subnet_cidr: "10.1.1.240/28"
+instance_name: nfs-server
+security_group_type: nfs_sg
```

Comment:
If this is going to be a general purpose role to deploy instances, it would be also useful to add the instance_type for example , so I can pick m4.large or m4.xlarge depending on the use case. I can see running this role with overrides such as instance_type , instance_name , etc...

Reply:
Good idea, I'll add this.